src/components/work/Header/Header.tsx

```diff
@@ -39,7 +39,7 @@ const Header = () => {
   const [isOpen, toggleNavbar] = React.useState<boolean>(false);
 
   return (
-    <header className="sticky-top bg-white" id="bee-main-header">
+    <header className="sticky-top bg-white" id="bee-main-header" style={{ 'minHeight': '4.125rem' }}>
```

Comment:
Here, a gap was sometimes showing up between the Header and SearchBar in mobile; root cause is that the header changes in height slightly. Adding an explicit `min-height` allows me to assume a constant height for other sticky elements elsewhere. (Using `position: sticky` instead of `position: fixed` was supposed to allow me to ignore that, but as soon as I need other `sticky` elements it becomes a concern again.)

Reply:
i think we have a `custom-header-height` class in the customStyles class we can use

Reply:
you can probably change the 4rem to `4.125rem` if you need it to be that large

Reply:
That. Helps. So. Much.

🙏